packages/starlight/package.json

```diff
@@ -2,7 +2,10 @@
   "name": "@astrojs/starlight",
   "version": "0.5.5",
   "description": "Build beautiful, high-performance documentation websites with Astro",
-  "scripts": {},
+  "scripts": {
+    "test": "vitest",
+    "test:coverage": "vitest run --coverage.enabled --coverage.all --coverage.reportsDirectory=./__coverage__"
```

Comment:
In theory you can configure Vitest’s `coverage` settings in `vitest.config.ts`, but I couldn’t get this to work, so resorted to CLI arguments here.

Reply:
Perhaps you need to use the `--coverage` CLI option, and configure the rest of the options in the config?

Reply:
Yeah, tried that, but it still wouldn’t pick them up. `coverage` config is not supported in a root `vitest.workspace.ts` only in `vitest.config.ts`, so I wondered if maybe using workspaces is the issue. Unless… can you have a root config file and then the workspace config too? 🤔 That’s the one permutation I didn’t try.

Reply:
YES! A root-level `vitest.config.ts` works! Nice.